core/src/main/java/hudson/model/UpdateCenter.java

```diff
@@ -131,7 +131,7 @@
  */
 @ExportedBean
 public class UpdateCenter extends AbstractModelObject implements Saveable, OnMaster {
-	
+
```

Comment:
Still a ton of gratuitous whitespace changes. Please make sure your IDE is configured to not automatically reindent code you are not actively editing.